docs/source/introduction.md

```diff
@@ -26,4 +29,38 @@ These projects are independent from the Opsn Source GIS Stack project and any is
 
 ## Overview Diagram
 
-![Overview Diagram](img/QGIS-Server-PG-Project-Design.png)
\ No newline at end of file
+![Overview Diagram](img/QGIS-Server-PG-Project-Design.png)
+
+
+The Open Source GIS Stack (OSGS) is a platform based on Docker, which is a container deployment environment. Containers are isolated application runtimes. They typically contain a minimal version of an operating system, typically Linux. In that minimal version of the operating system the containers will typically have one single task they are responsible for doing, for example running PostGIS or running Geoserver and so on. Each container has a very specific task and the design intent that you do not deploy mutliple services in a single container. You must deploy each service in its own container then orcherstrate those containers together. This is all managed using Docker and Docker Compose which is the orchestration environemnt for Docker. OSGS is simply a set of Docker orchestration routines to bring up different services in different containers, make them all talk together and present them as one cohesive architecture.
```

Comment:
Each container has a very specific task and the design intent

add 

Each container has a very specific task and the design intent is

Docker Compose

docker-compose